frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.unit.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ORDERS, SAMPLE_DATABASE } from "__support__/sample_database_fixture";
```

Comment:
As in the [Storybook file comment](https://github.com/metabase/metabase/pull/28776#discussion_r1121740394), it'd be awesome if we can avoid using the sample database fixture